CRM/CiviAwards/Form/AwardReview.php

```diff
@@ -170,9 +172,38 @@ private function getErrorMessage($fields) {
       return 'You can only view the reviews that you have added!';
     }
 
+    if ($this->isReviewFromSsp() && !$this->contactHasPanelAccessToAward()) {
+      return 'You are not a member of any panel on this Award';
```

Comment:
How multilingual support will work here? Just curious :)

Reply:
We are supposed to use the ts function in the template. `{ts}` Some word {/ts}`